### PR TITLE
Added a fix for normalizing imports from more than one level of parent modules (issue/2152)

### LINKED ISSUE #2152: Unrecoverable exception with ``  from ..import ccc``
Hello, 
I got ` ERROR: Unrecoverable exception thrown when parsing ./aaa/bbb/ccc.py! This should NEVER happen.`  

~~~bash
$ python --version
Python 3.11.4
$ isort --version

                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.12.0
$ cat /etc/debian_version 
trixie/sid
$ tree
.
└── aaa
    └── bbb
        └── ccc.py
~~~

Content of the offending file:

~~~bash
$ cat aaa/bbb/ccc.py 
from ..import bbb
~~~

My file is valid (or I just found a Python bug):

~~~bash
$ python -m aaa.bbb.ccc
$
~~~

But isort fails:

~~~bash
$ isort .
ERROR: Unrecoverable exception thrown when parsing ./aaa/bbb/ccc.py! This should NEVER happen.
If encountered, please open an issue: https://github.com/PyCQA/isort/issues/new
Traceback (most recent call last):
  File "/home/louis/.virtualenvs/311/bin/isort", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/main.py", line 1226, in main
    for sort_attempt in attempt_iterator:
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/main.py", line 1210, in <genexpr>
    sort_imports(  # type: ignore
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/main.py", line 93, in sort_imports
    incorrectly_sorted = not api.sort_file(
                             ^^^^^^^^^^^^^^
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/api.py", line 429, in sort_file
    changed = sort_stream(
              ^^^^^^^^^^^^
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/api.py", line 210, in sort_stream
    changed = core.process(
              ^^^^^^^^^^^^^
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/core.py", line 422, in process
    parsed_content = parse.file_contents(import_section, config=config)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/louis/.virtualenvs/311/lib/python3.11/site-packages/isort/parse.py", line 522, in file_contents
    if "," in import_string.split(just_imports[-1])[-1]:
                                  ~~~~~~~~~~~~^^^^
IndexError: list index out of range
~~~

Everything works if I replace ` from ..import bbb`   with `from .. import bbb` (notice the space that was added/removed between `..` and `import`).

Thanks for your work!
-- Louis

### PR DESCRIPTION
**Pull request by bp72**
Closes #2152

### CODE REVIEW

File: isort/identify.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from typing import Iterator, NamedTuple, Optional, TextIO, Tuple
 
-from isort.parse import _normalize_line, _strip_syntax, skip_line
+from isort.parse import _strip_syntax, normalize_line, skip_line
```
Comment (staticdev) [inline isort/identify.py:8]: Following the same logic, I would rename `_strip_syntax` to `strip_syntax`. It will be better if you could also do it ;)

**Review by staticdev — CHANGES_REQUESTED** [review]
Thanks for your PR @bp72 do you mind checking the suggestion I made?

**Comment by bp72** [comment]
@staticdev sorry, missed the comment :) It's renamed now.

**Review by staticdev — APPROVED** [review]

**Comment by staticdev** [comment]
Thanks @bp72, LGTM.

**Pull request merged**